netbox/templates/circuits/circuittermination.html

```diff
@@ -0,0 +1,125 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load i18n %}
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item"><a href="{% url 'circuits:circuit_list' %}?provider_id={{ object.provider.pk }}">{{ object.provider }}</a></li>
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col col-md-6">
+
+      <div class="card">
+            {% if object %}
+              <table class="table table-hover attr-table">
+                {% if object.site %}
+                  <tr>
+                    <th scope="row">{% trans "Site" %}</th>
+                    <td>
+                      {% if object.site.region %}
+                        {{ object.site.region|linkify }} /
+                      {% endif %}
+                      {{ object.site|linkify }}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th scope="row">{% trans "Termination" %}</th>
+                    <td>
```

Comment:
It looks like this block was mostly copied from `circuits/inc/circuit_termination.html`. We should move the content to a separate inclusion template to ensure it doesn't drift over time.